broker-cli/commands/info/index.js

```diff
@@ -24,14 +27,31 @@ module.exports = (program) => {
     .argument('<command>', '', Object.values(SUPPORTED_COMMANDS), null, true)
     .argument('[sub-arguments...]')
     .option('--rpc-address', 'Location of the RPC server to use.', validations.isHost)
+    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
     .action(async (args, opts, logger) => {
-      const { command } = args
+      const { command, subArguments } = args
+
+      let since
+      let limit
 
       switch (command) {
         case SUPPORTED_COMMANDS.SUPPORTED_MARKETS:
           return supportedMarkets(opts, logger)
+        case SUPPORTED_COMMANDS.TRADES:
+          [since, limit] = subArguments
```

Comment:
remove the `let since` && `let limit` from above and just use `const [since, limit] = subArguments